metrics/src/main/java/tech/pegasys/pantheon/metrics/prometheus/PrometheusMetricsSystem.java

```diff
@@ -48,9 +49,16 @@
   private static final String PANTHEON_PREFIX = "pantheon_";
   private final Map<MetricCategory, Collection<Collector>> collectors = new ConcurrentHashMap<>();
 
+  private static final Supplier<MetricsSystem> INSTANCE =
+      Suppliers.memoize(PrometheusMetricsSystem::init);
```

Comment:
I think this is wrong as it makes the metrics system JVM-wide whereas we want to be able to run two separate Pantheon instances in the same JVM and have them entirely isolated.

Reply:
My Prometheus Server code introduced the test breakage.  So this is reverted.